src/main/thrift/parquet.thrift

```diff
@@ -211,6 +204,93 @@ struct Statistics {
    4: optional i64 distinct_count;
 }
 
+/** Empty structs to use as logical type annotations */
```

Comment:
I think think it would help the reader if the comments for each logical type mention the physical type(s) they can annotate. Currently one needs to look them up in the table below, and then look in LogicalTypes.md or ConvertedType to find what physical types can be annotated.